### PR TITLE
adds python file and updates list files for running standard grid with r22.11.1

### PR DESCRIPTION
**Pull request by evanhanders**
See title. Let me know if it would be good to organize the repository structure differently, etc.

### CODE REVIEW

**Comment by mathren** [comment]
This would probably work fine, but I would suggest to move the script in `src/scripts`. (which would also require changing the hardcoded path to `MESA_input` in the script.) This allows us to use showyourwork `paths` module to go look for the MESA input directories instead of harcoding the path. There is no functional difference here just a matter of taste, I've pushed a commit that implements this.

**Comment by evanhanders** [comment]
Works for me! Wasn't sure how showyourwork required the repo structure.

I think Cole wants a few different metallicity grids included in the standard models, so let's hold off on merging this in for just a couple of days until that is finalized. Will change the generation script and template a bit.

**Comment by evanhanders** [comment]
By the way, I forgot to say this a few weeks back, but I think this PR for the standard models is ready to be checked and merged in!

**Comment by mathren** [comment]
Seems all good to me, @colej and @MichielsenM ?

File: src/data/MESA_input/inlist_standard
Comment (MichielsenM) [inline src/data/MESA_input/inlist_standard:1]: For the atmosphere I see that we have 
atm_T_tau_opacity = 'iterated'

I would usually set this to 'varying'. Any specific reason to use 'iterated' instead?

**Review by MichielsenM — COMMENTED** [review]
Looks alright to me, I only have a small remark about the atmosphere settings in the inlist

File: src/data/MESA_input/inlist_standard
Comment (evanhanders) [reply src/data/MESA_input/inlist_standard:1]: @MichielsenM  -- Nope! Is it better to set it to 'varying'? If so, happy to make the change (and re-run).  What's the difference?

Comment (MichielsenM) [reply src/data/MESA_input/inlist_standard:1]: @evanhanders  'Varying' is more physical, but probably takes a little bit more computation time (although not enough to worry about). The difference boils down to the following:
'Iterated' assumes the same opacity at each optical depth (uniform opacity in the atmosphere), but uses iterations to find an opacity consistent with the thermodynamic state at the base of the atmosphere. 
'Varying' evaluates the opacity consistent with the local thermodynamic state at each optical depth in the atmosphere via calls to MESA's EOS module.

Comment (evanhanders) [reply src/data/MESA_input/inlist_standard:1]: @MichielsenM great! I've modified this to 'varying' in the most recent commit and I'm re-running the grid.

Comment (MichielsenM) [reply src/data/MESA_input/inlist_standard:1]: @evanhanders Then it seems all good to me to merge the branch :)

**Pull request merged**